lib/evaluation/handlers/subjectIsHolderEvaluationHandler.ts

```diff
@@ -165,16 +164,12 @@ export class SubjectIsHolderEvaluationHandler extends AbstractEvaluationHandler
     inputDescriptorIds: Array<string>
   ) {
     const entry = this.getValue(fieldIdzInputDescriptors, searchableFieldIds);
-    if (entry) {
-      searchableFieldIds.forEach((searchableFieldId) => entry.mappedFieldIds.add(searchableFieldId));
-      inputDescriptorIds.forEach((inputDescriptorId) => entry.mappedInputDescriptorIds.add(inputDescriptorId));
-    }
+    searchableFieldIds.forEach((searchableFieldId) => entry?.mappedFieldIds.add(searchableFieldId));
```

Comment:
Actually in this case the previous code was better, as right now you will start iterating all fields and descriptors if entry is undefined, which is unnecessary because it will do nothing.